variables.tf

```diff
@@ -646,7 +646,7 @@ variable "rds_iam_database_authentication_enabled" {
 variable "rds_parameter_group_name" {
   description = "Parameter group for database"
   type        = string
-  default     = ""
+  default     = "default"
```

Comment:
We could've also tested for an empty string.

Reply:
I think "default" nails it, as the default group is named by this as well

Reply:
@cytopia thought about this, too. I don't think it's a good idea to try and mimic the resource names that AWS is handing out themselves. It's always good to have a good distinction between the two, even though the config is identical to the **real** default. Luckily we can look at the tags to see if it's TF that's handling that.

I'll give you an example: the special case of the default VPC.

Reply:
As always this can be improved, or now it was to solve a problem that in the future will be addressed by the module itself once they merge the changes(if they ever do it)